product-web-app/views/add-band.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <title>add Band</title>
+    <link rel='stylesheet' href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
+    <script>
+        function showConfirmation() {
+            alert("Data added correctly!");
+        }
+    </script>
+</head>
+<body>
+    <header class="bg-dark text-light">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-12 py-3">
+                    <h1>Band</h1>
+                    <nav>
+                        <ul class="nav">
+                        </ul>
+                    </nav>
+                </div>
+            </div>
+        </div>
+    </header>
+    <main class="container mt-4">
+        <section id="form" class="bg-light py-3">
+            <div class="container">
+                <h2>Enter Band details</h2>
+                <form action="/admin/add-band" method="post">
+                <div class="form-group">
+                    <label for="name">Name</label>
+                    <input type="text" id="name" name="name" class="form-control" value="{{ name }}" required>
+                </div>
+                <div class="form-group">
+                    <label for="level">Level</label>
+                    <input type="text" id="level" name="level" class="form-control" value="{{ level }}" required>
+                </div>
+                <div class="form-group">
+                    <label for="responsibilities">Responsibilities</label>
+                    <input type="text" id="responsibilities" name="responsibilities" class="form-control" value="{{ responsibilities }}" required>
+                </div>
+                <button type="submit" class="btn btn-primary" onclick="showConfirmation()">Submit</button>
```

Comment:
onclick="showConfirmation() that will result in confirmation message even if it's gonna fail on backend. You need to add a different way - have a look at how errors were shown in workshops and update it for success messages